domain/entity/latest_entry.go

```diff
@@ -0,0 +1,60 @@
+package entity
+
+import (
+	"time"
+
+	"google.golang.org/appengine/datastore"
+)
+
+const (
+	LatestEntryCodeTamai    = "tamai-sd"  // LatestEntryCodeTamai defines Shiori Tamai blog code
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const LatestEntryCodeTamai should have comment (or a comment on this block) or be unexported